package.json

```diff
@@ -85,9 +91,21 @@
       ]
     },
     "angular-mock": "global:angular.mock",
-    "diff-match-patch": "diff_match_patch",
-    "dom-text-mapper": "DomTextMapper",
-    "dom-text-matcher": "DomTextMatcher",
+    "dom-anchor-fragment": "domAnchorFragment",
```

Comment:
Not sure I understand. Aren't we requiring the unbundled versions of these? At which point it's all up to browserify?